.github/workflows/close-empty-issues.yml

```
@@ -0,0 +1,13 @@
+on:
+  issues:
+    types: [opened]
+jobs:
+  closeEmptyIssues:
+    if: "${{ github.event.issue.body == '' }}"
+    runs-on: ubuntu-latest
+    steps:
+      - name: close empty issues
+        uses: kerhub/saved-replies@v0.3.0
```

Let's use the actual hash here:

```suggestion
        // v0.3.0
        uses: kerhub/saved-replies@0123456789abcdef
```